docs/about-stacks.md

```diff
@@ -24,47 +24,48 @@ A terraform stack is a runnable terraform module that operates on a subset of
 the infrastructure's resources.
 ```
 
-By _runnable_ it means it has everything needed to call
-`terraform init/plan/apply` . For example, a module used by other modules, ie,
-don't have the `provider` explicitly set, is not runnable hence it's
-**not a stack**.
+For a module to be considered a stack, it must be complete and self-contained, 
+meaning that it can run all the necessary Terraform commands (`init/plan/apply`) 
+without any external dependencies.
 
-If your runnable Terraform module creates your whole infrastructure, *it's
-also not a stack*, since the idea of stacks is to be a unit of infrastructure
-decomposition.
+If a module is missing any required components, such as provider settings, 
+it cannot be considered a stack.
 
-## Isolate code with higher change frequency
+It's also worth noting that if a module creates your entire infrastructure, 
+it's not a stack. **Stacks are designed to break down your infrastructure into** 
+**manageable pieces**, so they operate on a specific subset of resources rather 
+than your entire infrastructure.
 
-If you have a high change frequency in your infrastructure, for example, several
-deploys per day/week or several configuration changes per day/week, then if you
-apply the change in your single Terraform project, some dependent resources can
-be recreated leading to increased downtime. There are several reasons why a
-dependent module can be recreated, eg.: attributes with variable interpolation
-from recreated resources; underspecified attributes refreshing during plan, etc.
+## Isolate code that changes frequently
 
-By using stacks you can modify only the stacks affected by the deploys or
-configuration changes needed, but you have to choose the size of the stack
-wisely to avoid duplication.
+If you make a lot of updates to your infrastructure, like deploying new code or changing 
+settings often, doing everything in one Terraform project could cause problems. 
+This might lead to some parts of your infrastructure needing to be rebuilt, which means 
+more downtime.
 
-The overall principle being, code that changes at different rate change for
-different reasons and code that changes for different reasons should be as
-isolated from each other as possible.
+If you frequently update your infrastructure, such as deploying new code or modifying settings, 
+doing everything into one Terraform project could cause complications. This might lead to 
+some parts of your infrastructure needing to be rebuilt, which means increased downtime.
```

Comment:
this phrase is repeated with different wording..

Reply:
> this phrase is repeated with different wording..

@i4ki what's your suggestion in that case? Shall we revert this to the previous version or would you like to suggest a more clear wording?

Reply:
I meant that it's duplicated, sorry

Reply:
```
If you make a lot of updates to your infrastructure, like deploying new code or changing settings often, doing everything in one Terraform project could cause problems. This might lead to some parts of your infrastructure needing to be rebuilt, which means more downtime.

If you frequently update your infrastructure, such as deploying new code or modifying settings, doing everything into one Terraform project could cause complications. This might lead to some parts of your infrastructure needing to be rebuilt, which means increased downtime.
```

See here: https://github.com/mineiros-io/terramate/blob/b9345b6258b8ac37d956a03505adb1a610a363eb/docs/about-stacks.md#isolate-code-that-changes-frequently

Reply:
that makes sense - thank you